From 29dc7fa4f71efbaee688db21775d0a0ec1a51ef3 Mon Sep 17 00:00:00 2001
From: Yasir Khan <yasir_electronics@yahoo.com>
Date: Mon, 21 Aug 2017 22:43:28 +0800
Subject: [PATCH 1/3] usb/serial: add vid/pid of Quectel EC20 GSM modem

Add vid/pid of Quectel EC20 modem so that usb serial
device is created for it.

Signed-off-by: Yasir Khan <yasir_electronics@yahoo.com>
---
 drivers/usb/serial/option.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/usb/serial/option.c b/drivers/usb/serial/option.c
index 4021846..948eec4 100644
--- a/drivers/usb/serial/option.c
+++ b/drivers/usb/serial/option.c
@@ -624,6 +624,8 @@ static const struct option_blacklist_info sierra_mc73xx_blacklist = {
 };
 
 static const struct usb_device_id option_ids[] = {
+	/* For Quectel EC20 GSM modem */
+	{ USB_DEVICE(0x05C6, 0x9215) },
 	{ USB_DEVICE(OPTION_VENDOR_ID, OPTION_PRODUCT_COLT) },
 	{ USB_DEVICE(OPTION_VENDOR_ID, OPTION_PRODUCT_RICOLA) },
 	{ USB_DEVICE(OPTION_VENDOR_ID, OPTION_PRODUCT_RICOLA_LIGHT) },
-- 
1.9.1

